electron/walletBackup/gdrive/index.js

```diff
@@ -0,0 +1,144 @@
+import EventEmitter from 'events'
+import config from 'config'
+import { forwardEvent } from '@zap/utils/events'
+import { mainLog } from '@zap/utils/log'
+import createClient from './gdrive'
+
+class BackupService extends EventEmitter {
+  drive = null
+
+  constructor() {
+    super()
```

Comment:
constructor can be removed completely if the only thing it is doing is calling `super()`

Reply:
This is an actually interesting case. The reason i've added it was `drive = null` instance variable, which I thought translates into something like
```constructor() {